### PR TITLE
[release-4.7] Bug 2067699: Update to moby v20.10.14

### PR DESCRIPTION
**Pull request by bertinatto**
CC @openshift/storage

### CODE REVIEW

File: vendor/sigs.k8s.io/sig-storage-local-static-provisioner/pkg/deleter/deleter.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ import (
 	"sigs.k8s.io/sig-storage-local-static-provisioner/pkg/common"
 	"sigs.k8s.io/sig-storage-local-static-provisioner/pkg/metrics"
 
-	v1 "k8s.io/api/core/v1"
```
Comment (bertinatto) [inline vendor/sigs.k8s.io/sig-storage-local-static-provisioner/pkg/deleter/deleter.go:32]: This was changed by `go mod vendor` (1.15.15).

**Comment by dobsonj** [comment]
> /test e2e-operator

CI is completely broken on 4.7 right now :(
See https://bugzilla.redhat.com/show_bug.cgi?id=2059513

**Pull request merged**